src/dotnet/APIView/APIViewWeb/Account/ResolverRequirementHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Configuration;
+
+namespace APIViewWeb
+{
+    public class ResolverRequirementHandler : IAuthorizationHandler
```

Comment:
One alternate option is to inherit ApproverRequirementHandler and make approver hashset as protected. But given current time line, it's fine to do and we can optimize it later.